vizro-core/src/vizro/models/types.py

```diff
@@ -81,32 +84,61 @@ def __init__(self, function, /, *args, **kwargs):
 
         # Since we do __call__ as self.__function(**bound_arguments.arguments), we need to restructure the arguments
         # a bit to put the kwargs in the right place.
-        # For a function with parameter **kwargs this converts self.__arguments = {"kwargs": {"a": 1}} into
-        # self.__arguments = {"a": 1}.
-        if var_keyword_param in self.__arguments:
-            self.__arguments.update(self.__arguments[var_keyword_param])
-            del self.__arguments[var_keyword_param]
-
-    def __call__(self, **kwargs):
-        """Run the `function` with the initial arguments overridden by **kwargs.
-
-        Note *args are not possible here, but you can still override positional arguments using argument name.
+        # For a function with parameter **kwargs this converts self.__bound_arguments = {"kwargs": {"a": 1}} into
+        # self.__bound_arguments = {"a": 1}.
+        if var_keyword_param in self.__bound_arguments:
+            self.__bound_arguments.update(self.__bound_arguments[var_keyword_param])
+            del self.__bound_arguments[var_keyword_param]
+
+    def __call__(self, *args, **kwargs):
```

Comment:
Given that we have worked on this before, why was there another change needed? It doesn't seem to appear in the PR description